test/run-drun/try-finally-trap/try-finally-trap.mo

```diff
@@ -0,0 +1,24 @@
+import { debugPrint; error; call_raw; principalOfActor; rts_callback_table_count } =  "mo:⛔";
+
+actor A {
+
+  public func go() : async () {
+    try {
+      await async ();
+      assert false
+    }
+    finally {
+      debugPrint("trap in finally!");
```

Comment:
Then I expect to see this output twice.

Reply:
Don't follow, why?

Reply:
OMG, you are right. But that's weird.  I guess we execute the finally on the success path, it traps and rolls back and then executes the finally a second time in the cleanup. I guess that's benign except for the logging....

Reply:
Right. Nothing to be ashamed of...